plugins/GSdx/GSRendererDX.cpp

```diff
@@ -444,6 +446,54 @@ void GSRendererDX::DrawPrims(GSTexture* rt, GSTexture* ds, GSTextureCache::Sourc
 		}
 	}
 
+	// Upscaling hack to avoid various line/grid issues
```

Comment:
Would it be be possible to move the code in a common function in the HW renderer ?